arm-network/2017-06-01/swagger/applicationGateway.json

```diff
@@ -384,6 +384,84 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Network/applicationGatewayAvailableSslOptions/default": {
```

Comment:
We'd encourage you to please provide [x-ms-examples](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/x-ms-examples.md#why-x-ms-examples) for the newly introduced operations to help us with model validation.